rholang/src/test/scala/coop/rchain/rholang/interpreter/CostAccountingReducerTest.scala

```diff
@@ -94,7 +94,7 @@ class CostAccountingReducerTest extends FlatSpec with Matchers with TripleEquals
     assert(test === Left(OutOfPhlogistonsError))
   }
 
-  it should "stop interpreter threads as soon as deploy runs out of phlo" in {
+  it should "stop interpreter threads as soon as deploy runs out of phlo" ignore {
```

Comment:
should this stay ignored?

Reply:
I'm afraid this will have to be ignored until @JosDenmark 's https://rchain.atlassian.net/browse/RCHAIN-3504 is in